contracts/core/Registry.sol

```diff
@@ -2,17 +2,18 @@
 pragma solidity 0.8.19;
 
 // External Libraries
-import "openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
-import "openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
+import {AccessControlUpgradeable} from
+    "openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
+import {Initializable} from "openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
 import {ERC20} from "solady/tokens/ERC20.sol";
 // Interfaces
-import "./interfaces/IRegistry.sol";
+import {IRegistry} from "./interfaces/IRegistry.sol";
```

Comment:
(This can be in another PR) Would be cool to remove relative paths and just use the remappings.

Reply:
100% I started changing but it was popping up more issues on certain contracts so I left for an other PR the rest. I will open a ticket